tola/locale/es/LC_MESSAGES/djangojs.po

```diff
@@ -48,50 +44,28 @@ msgstr[1] ""
 "Eliminar este objetivo implica que %s resultados no tedrán más objetivos "
 "asociados."
 
-#. Translators: Button to approve a form
-#: js/base.js:415 js/base.js:517 js/base.js:566 js/base.js:609
-#: js/components/changesetNotice.js:65
-msgid "Ok"
-msgstr "OK"
-
-#. Translators: Button to cancel a form submission
-#: js/base.js:417 js/base.js:519 js/base.js:568 js/base.js:611
-#: js/components/changesetNotice.js:67
-#: js/pages/results_framework/components/level_cards.js:578
-msgid "Cancel"
-msgstr "Cancelar"
-
-#: js/base.js:462 js/components/changesetNotice.js:173
-msgid "A reason is required."
-msgstr "Se requiere una justificación."
-
-#: js/base.js:526 js/base.js:621
-#: js/pages/results_framework/components/level_cards.js:89
-#: js/pages/tola_management_pages/country/models.js:446
-msgid "This action cannot be undone."
-msgstr "Advertencia: esta acción no se puede deshacer."
+#. Translators: the header of an alert after an action completed successfully
+#: js/base.js:388
+msgid "Success"
+msgstr "Éxito"
 
-#: js/base.js:530 js/base.js:625
-#: js/pages/results_framework/components/level_cards.js:86
-#: js/pages/results_framework/models.js:665
+#. Translators: the header of an alert where additional warning info is provided
+#: js/base.js:407 js/pages/results_framework/components/level_cards.js:86
+#: js/pages/results_framework/models.js:620
 #: js/pages/tola_management_pages/country/components/edit_disaggregations.js:626
-#: js/pages/tola_management_pages/country/models.js:444
-#: js/pages/tola_management_pages/country/models.js:479
-#: js/pages/tola_management_pages/country/models.js:511
-#: js/pages/tola_management_pages/program/models.js:277
+#: js/pages/tola_management_pages/country/models.js:434
+#: js/pages/tola_management_pages/country/models.js:469
+#: js/pages/tola_management_pages/country/models.js:501
+#: js/pages/tola_management_pages/program/models.js:279
 msgid "Warning"
 msgstr "Advertencia"
 
-#. Translators: This is a label for a dropdown that presents several possible justifications for changing a value
-#: js/base.js:575 js/components/changesetNotice.js:19
-#: js/pages/results_framework/components/level_cards.js:404
-#: js/pages/tola_management_pages/audit_log/views.js:235
-msgid "Reason for change"
-msgstr "Razón para el cambio"
-
-#: js/base.js:687
-msgid "Success!"
-msgstr "¡Éxito!"
+#. Translators: the header of an alert after an action failed for some reason
+#: js/base.js:425
+#, fuzzy
```

Comment:
Looks like there's a fuzzy translation that should be addressed before compiling.